### DISCUSSION TITLE
Editable, copy or paste location into or out of Edit Link --> Browse --> "Select File" dialog

### DISCUSSION THREAD

**Discussion by tscahill** [zim-desktop-wiki/zim-desktop-wiki#2357] [Q&A] [answered]
The "Edit Link" dialog has fully editable fields. Would it be possible to implement the same in the subsequent "Select File" dialog? If so, how difficult would it be?

I searched the discussions for the word "Browse" and nothing seemed to match. Sorry if I missed something obvious.

**Comment by introt** [comment, marked as answer]
You can press the standard (file) browser shortcut `Ctrl` + `L` to access the location bar. This allows you to type or paste in the path of the file you'd like to link to.

**Reply by tscahill** [reply]
Awesome! Thanks introt. (How did I not know this key combo?)
